### PR TITLE
Fixed errcheck issues in server/channels/app/server_test.go

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29089
Jira https://mattermost.atlassian.net/browse/MM-61491
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: server/channels/app/server_test.go [hunk 1]
```diff
@@ -362,7 +367,8 @@ func TestPanicLog(t *testing.T) {
 	}
 
 	client := &http.Client{Transport: tr}
-	client.Get("https://localhost:" + strconv.Itoa(s.ListenAddr.Port) + "/panic")
+	_, err = client.Get("https://localhost:" + strconv.Itoa(s.ListenAddr.Port) + "/panic")
+	require.NoError(t, err)
```
Comment (hanzei) [inline server/channels/app/server_test.go:371]: This is allowed to error.

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR :+1:

**Comment by Aryakoste** [comment]
@hanzei Changes done

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Pull request merged**